### ISSUE TITLE
Quicklook preview

### ISSUE THREAD

**Issue by mayurankv** [chocoford/ExcalidrawZ#24]
Does this app give a quicklook preview for excalidraw? If not would it be possible to?

**Comment by chocoford** [comment]
Hello, @mayurankv.
Do you mean to perform a Quick Look preview operation on Excalidraw files?

**Comment by mayurankv** [comment]
Yep! As in when you press space on a file in finder. I tested this and I don't think the app currently does this, but I would have thought it would (hopefully) not be too much work to add this, and it would be extremely useful! For example, when using multiple diagrams, knowing which diagram is which without having to open them from finder would be handy!

I unfortunately don't have much experience with swift or mac apps in particular currently so I'm not sure I'd be able to contribute unfortunately, but I believe it would involve following this page: https://developer.apple.com/documentation/quicklook

**Comment by dalisoft** [comment]
@chocoford macOS Quicklook app demo https://www.ipreview.app. But for excalidraw files supported by your app
